From d9554495b54d1f150a85f4e329412ff62728d9c7 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 15 Feb 2024 12:06:44 +0000
Subject: [PATCH] Revert "chore: use git fork for aws-nitro-enclaves-cose"

This reverts commit 1ea09e88baee3c47cdc76902f0034d553207d515.
---
 data-formats/Cargo.toml | 2 +-
 http-wrapper/Cargo.toml | 2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

Sdiff --git a/data-formats/Cargo.toml b/data-formats/Cargo.toml
index 714fc55..6d00eb4 100644
--- a/data-formats/Cargo.toml
+++ b/data-formats/Cargo.toml
@@ -17,7 +17,7 @@ serde_cbor = "0.11"
 serde_repr = "0.1.6"
 serde_tuple = "0.5"
 thiserror = "1"
-aws-nitro-enclaves-cose = { git = "https://github.com/nullr0ute/aws-nitro-enclaves-cose/", rev = "e3938e60d9051690569d1e4fcbe1c0c99d2fafa8" }
+aws-nitro-enclaves-cose = "0.4.0"
 uuid = "1.3"
 num-traits = "0.2"
 num-derive = "0.3"
diff --git a/http-wrapper/Cargo.toml b/http-wrapper/Cargo.toml
index 5d4999c..7b98097 100644
--- a/http-wrapper/Cargo.toml
+++ b/http-wrapper/Cargo.toml
@@ -20,7 +20,7 @@ openssl = "0.10.60"
 
 fdo-data-formats = { path = "../data-formats", version = "0.5.0" }
 fdo-store = { path = "../store", version = "0.5.0" }
-aws-nitro-enclaves-cose = { git = "https://github.com/nullr0ute/aws-nitro-enclaves-cose/", rev = "e3938e60d9051690569d1e4fcbe1c0c99d2fafa8" }
+aws-nitro-enclaves-cose = "0.4.0"
 
 # Server-side
 uuid = { version = "1.3", features = ["v4"], optional = true }
-- 
2.43.1

